R/utils.R

```diff
@@ -64,16 +64,26 @@ create_item <- function(pkg, type = c("requirements","test_cases","test_code"),
 #'
 #' Wrapper for whoami::username
 #'
+#' @param pkg path to base directory of working project
+#'
 #' @returns `[character]` Username of the person that called the function
 #'
 #' @importFrom whoami username
 #' @export
 #' @examples
+#' \dontrun{
```

Comment:
We might want to use a temp directory here instead of dontrun. CRAN has yelled at me in the past for using dontrun, they prefer if you have runnable code unless you're hitting some external resource that wouldn't be available in a CRAN check.

Reply:
I've been yelled at for this too, at this time I was focused on getting the updates done. haha. we can update them. part of it is that these functions need so much set up that it can be a lot since they are dependent on there being the validation infrastructure.